### PR TITLE
feat: Add timer icon

### PR DESCRIPTION
**Pull request by ahtohbi4**
Icon "timer" was added.

Icon timer

### CODE REVIEW

**Comment by ericfennis** [comment]
Nice, love it! 🔥

**Review by ericfennis — DISMISSED** [review]

File: tags.json [hunk 1]
```diff
@@ -209,6 +209,7 @@
   "thermometer": ["temperature", "celsius", "fahrenheit", "weather"],
   "thumbs-down": ["dislike", "bad", "emotion"],
   "thumbs-up": ["like", "good", "emotion"],
+  "timer": ["time", "timer"],
```
Comment (ericfennis) [inline tags.json:212]:
```suggestion
  "timer": ["time", "timer", "stopwatch"],
```

**Comment by locness3** [comment]
I think the middle part could be bigger, even if that requires not respecting the 2px safe zone rule

**Comment by ericfennis** [comment]
@locness3 Well spotted.

**Review by ericfennis — COMMENTED** [review]
@locness3 with 2px spacing 🤔 
image
image

**Comment by locness3** [comment]
Right looks good.

File: icons/timer.svg [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <line x1="10" x2="14" y1="2" y2="2" />
+  <line x1="12" x2="15" y1="13" y2="10" />
+  <circle cx="12" cy="13" r="8" />
```
Comment (ericfennis) [inline icons/timer.svg:14]: Add 2px spacing.
```suggestion
  <line x1="12" x2="15" y1="14" y2="11" />
  <circle cx="12" cy="14" r="8" />
```

**Pull request merged**